vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/PathAuthorizationValidator.kt

```diff
@@ -0,0 +1,27 @@
+package at.asitplus.wallet.lib.agent
+
+import at.asitplus.jsonpath.core.NormalizedJsonPath
+
+interface PathAuthorizationValidator {
```

Comment:
Whats the advantage over using a `(SubjectCredentialStore.StoreEntry, NormalizedJsonPath) -> Boolean` declaration for a parameter whenever this interface is used (probably as a type alias)?

Reply:
This allows using named parameters on function invocation 

changed invocation to actually use named parameters after this comment, in commit c1c70da607a02172e020331bea8b0f98e95eb3f3

Reply:
I'd argue that using a function type is (1) more aligned with our other parameters (e.g in `OidcSiopWallet` ... although I spot a `fun interface` there too) and (2) decouples the implementation from the client somewhat more, as argued in Java to Kotlin chaptler 16 (see https://github.com/java-to-kotlin/book/blob/400cc56b7078b5bf8258b4d309093d0002a09493/text/src/chapters/16-interfaces-to-functions.asciidoc)
See also https://github.com/a-sit-plus/kmm-vc-library/commit/3b0ff904b44cdf41c168aab12899d1951e77c8df